azure api management was connected to func, tests on apim pass, func fails with 404.
I don't think this is a code issue, seems to just be an issue with some connection that I'm not seeing. 

